zilsd.adoc

```diff
@@ -39,9 +39,17 @@ Even if naturally aligned, the memory access might not be performed atomically.
 
 If the effective address is a multiple of 4, then each word access is required to be performed atomically.
 
-To ensure fault handling is possible for the load instructions, it must be ensured that the register which is the source of the base address is not overwritten before the entire operation is complete.
+The following table summarizes the required behavior:
+
+|===
+|Alignment |Word accesses guaranteed atomic? |Can cause misaligned trap?
+|8B |yes |no
+|4B not 8B |yes |yes
+|else |no | yes
+|===
+
+To ensure resumable trap handling is possible for the load instructions, if a trap is taken, the base register must have its original value, while the other register in the pair can have been updated.
```

Comment:
Is the last part of this sentence (after the last comma) necessary? It's currently a lot of commas, and the sentence is quite long, making it a bit hard to read. 
Maybe just cut it down to something like this? "To ensure resumable trap handling is possible for the load instructions, if a trap is taken the base register must keep its original value".

Reply:
I have simplified by splitting into two sentences. The last part is however important to completely specify the behavior

Reply:
Looks good 👍